cmd/snap-bootstrap/cmd_initramfs_mounts_test.go

```diff
@@ -94,6 +95,21 @@ var (
 		DevNum:            "default",
 	}
 
+	defaultBootWithSaveDisk = &disks.MockDiskMapping{
+		FilesystemLabelToPartUUID: map[string]string{
+			// ubuntu-boot not strictly necessary, since we mount it first we
+			// don't go looking for the label ubuntu-boot on a disk, we just
+			// mount it and hope it's what we need, unless we have UEFI vars or
+			// something
+			"ubuntu-boot": "ubuntu-boot-partuuid",
+			"ubuntu-seed": "ubuntu-seed-partuuid",
+			"ubuntu-data": "ubuntu-data-partuuid",
+			"ubuntu-save": "ubuntu-save-partuuid",
+		},
+		DiskHasPartitions: true,
+		DevNum:            "default",
```

Comment:
nitpick, could you make this something different from the mock disk mapping above it like `default-with-save`?